buildbot/dependency.conf

```diff
@@ -5,7 +5,7 @@ ocl_cpu_rt_ver=2020.10.6.0.04
 ocl_cpu_rt_ver_win=2020.10.6.0.04
 # Same GPU driver supports Level Zero and OpenCL:
 # https://github.com/intel/compute-runtime/releases/tag/20.21.16886
-ocl_gpu_rt_ver=20.21.16886
+ocl_gpu_rt_ver=20.26.17199
```

Comment:
please update link above when you change driver version.

Reply:
done